### PR TITLE
General slime improvements

### PR DESCRIPTION
**Pull request by Just-a-Unity-Dev**
## About the PR

#23354 - Companion issue regarding slime balance

This PR will attempt to add the following:

- [x] Slimes can "morph" into a smaller slime which is faster and can go through grilles, but drops all their items.
- [x] Slimes have an internal 2x2 inventory - you can examine a slime to see whether they have items in their inventory, and can grab it.
- [ ] ~~Same damage mods however less bloodloss damage~~ mods are boring
- [x] Give slimes a bit lower punch speed but increase punch damage (i'm fairly sure getting punched by a bag of concrete would really hurt)
- [x] Slimes have an accelerated regen (slight)
- [x] Morphed slimes are different from NPC slimes because they have proper transparency
- [x] Morphs can be put into duffelbags and take up the entire slot

## Why / Balance

Due to slimes having all negatives, I have decided to make positives for them. 

## Technical details

swaps around some entities ids

## Media

- [x] I have added screenshots/videos to this PR showcasing its changes ingame, **or** this PR does not require an ingame showcase

## BUGFUL FOOTAGE - WILL CHANGE ON PR FINISH

https://github.com/space-wizards/space-station-14/assets/67359748/ea58f7ac-abf2-4bdd-985a-258752cc948f

https://github.com/space-wizards/space-station-14/assets/67359748/e8e8a0b1-b881-47c6-97d1-3eb8fcf504ba

## Breaking changes

**Changelog**

:cl: eclips_e
- add: Slimepeople can now morph into a "geras"--a smaller slime form that can pass under grilles, at the cost of dropping all of their inventory. They can also be picked up with two hands and placed into duffelbags.
- add: Slimepeople now have an internal 2x2 storage that they (and anyone around them) can access. It is not dropped when morphing into a geras!
- add: Slimepeople now have slightly increased regeneration and a slightly meatier punch, but slower attacks.

### CODE REVIEW

**Comment by K-Dynamic** [comment]
Ekrixi dev when

**Comment by mirrorcult** [comment]
based

**Comment by BreadyToad** [comment]
Morphing part is cool, but a bit too unbalanced. It would be better if slimes could change their hairstyle and beard as they wish and could change colour with spray painter (since we don't have paint, like in ss13) overall original slimepeople bonuses came from them being able to regen lost limbs and having no bones, so adding this would result in a very powerful race later on when surgery gets added (if it ever does)

**Comment by Just-a-Unity-Dev** [comment]
> Morphing part is cool, but a bit too unbalanced. It would be better if slimes could change their hairstyle and beard as they wish and could change colour with spray painter (since we don't have paint, like in ss13). Overall original slimepeople bonuses came from them being able to regen lost limbs and having no bones, so adding this would result in a very powerful race later on when surgery gets added (if it ever does)

I can nerf them to being only slightly faster and unable to crowbar doors but I insist on the morphing as its a really interesting mechanic

**Comment by K-Dynamic** [comment]
Where is the tg self-revival mechanic 😢 

https://github.com/space-wizards/space-station-14/issues/23354#issuecomment-1873701892

Also idk about crowbaring doors faster, doesn't seem very 'slimy' to me (feels more like a diona thing)

**Comment by BreadyToad** [comment]
> > Morphing part is cool, but a bit too unbalanced. It would be better if slimes could change their hairstyle and beard as they wish and could change colour with spray painter (since we don't have paint, like in ss13). Overall original slimepeople bonuses came from them being able to regen lost limbs and having no bones, so adding this would result in a very powerful race later on when surgery gets added (if it ever does)
> 
> I can nerf them to being only slightly faster and unable to crowbar doors but I insist on the morphing as its a really interesting mechanic

Yeah, that would probably work. My main concern was the antag factor, with thief slimes being able to outrun secoffs with stolen item, but as long as they are slower than speedboots and faster than walking speed both slimes get a cool mechanic and station gets a reason to produce speedboots. Morphs would probably need to have no hands though, and it would be cool if they looked like mini versions of the mob slimes

**Comment by deltanedas** [comment]
that bug was just a typo using PartBase not BasePart

**Comment by icekot8** [comment]
Ohhhh. What's the maximum size of a morphed slime's inner bag? We all realize they're going to put cigarette packs full of pills (or a spacesuit RD) in there, don't we ?
What do you mean put in the bag? The morphed version or the full-sized version? How does the antagonist kill the slime targets? How does the security team solve the problem? This seems wrong...

**Comment by deltanedas** [comment]
a slime morphing would drop all equipment so its not op for thieves since they drop everything they have for an item or 2

**Comment by Just-a-Unity-Dev** [comment]
> Ohhhh. What's the maximum size of a morphed slime's inner bag? We all realize they're going to put cigarette packs full of pills (or a spacesuit RD) in there, don't we ? What do you mean put in the bag? The morphed version or the full-sized version? How does the antagonist kill the slime targets? How does the security team solve the problem? This seems wrong...

An internal inventory would be 2x2.

Also, I mean that you can place slimes inside of bags, such as duffelbags/satchels... cause I think it would be funny as hell to carry a friend slime in a duffelbag

> Yeah, that would probably work. My main concern was the antag factor, with thief slimes being able to outrun secoffs with stolen item, but as long as they are slower than speedboots and faster than walking speed both slimes get a cool mechanic and station gets a reason to produce speedboots. Morphs would probably need to have no hands though, and it would be cool if they looked like mini versions of the mob slimes

Morphed slimes wouldn't get an inventory except for their internal one

**Comment by K-Dynamic** [comment]
> Also, I mean that you can place full-sized slimes inside of bags, such as duffelbags/satchels... cause I think it would be funny as hell to carry a friend slime in a duffelbag

Might be annoying to be suddenly picked up and put into a bag if this applies while full-sized

**Comment by Just-a-Unity-Dev** [comment]
> > Also, I mean that you can place full-sized slimes inside of bags, such as duffelbags/satchels... cause I think it would be funny as hell to carry a friend slime in a duffelbag
> 
> Might be annoying to be suddenly picked up and put into a bag if this applies while full-sized

Would probably have a doafter of like 5-7 seconds

**Comment by icekot8** [comment]
> > Ohhhh. What's the maximum size of a morphed slime's inner bag? We all realize they're going to put cigarette packs full of pills (or a spacesuit RD) in there, don't we ? What do you mean put in the bag? The morphed version or the full-sized version? How does the antagonist kill the slime targets? How does the security team solve the problem? This seems wrong...
> 
> An internal inventory would be 2x2.
> 
> Also, I mean that you can place full-sized slimes inside of bags, such as duffelbags/satchels... cause I think it would be funny as hell to carry a friend slime in a duffelbag

That's more of a downside, bro. They pick you up, stick you in the microwave for a moment like you're a mouse, and that's it, you're instantly dead, you can't be brought back to normal. Similarly, they pick you up, throw you in the trash....... If you do a carry option, the slimes must be of impressive size and slow the carry, otherwise it will look like a cover.

**Comment by BreadyToad** [comment]
> > Ohhhh. What's the maximum size of a morphed slime's inner bag? We all realize they're going to put cigarette packs full of pills (or a spacesuit RD) in there, don't we ? What do you mean put in the bag? The morphed version or the full-sized version? How does the antagonist kill the slime targets? How does the security team solve the problem? This seems wrong...
> 
> An internal inventory would be 2x2.
> 
> Also, I mean that you can place full-sized slimes inside of bags, such as duffelbags/satchels... cause I think it would be funny as hell to carry a friend slime in a duffelbag

This shouldn't be applied to sull-sized ones due to their height and overall grief possibilities, especially with SSD targets. Plus, you can still carry around a morphed friend who could (I presume) talk and look out of your inventory

**Comment by lzk228** [comment]
imo, internal inventory and putting into bags is OP

**Comment by Just-a-Unity-Dev** [comment]
> imo, internal inventory and putting into bags is OP

It's a 2x2 inventory that drops on crit, if necessary I can make it so that people can open the internal inventory - not sure how bags are OP, but I guess I can make it so that only the morph can fit inside of a bag

**Comment by Kadeo64** [comment]
IMO this PR just seems to powercreep slimes massively. 
1 seems like a very OP ability used exclusively by shitters, to bust into a locked room and shoot someone dead instantly
2 seems fine honestly, it's the species ability I would give to slimes. Just make it so that examining them makes it obvious they have something inside.
3 and 4 are fine, although I can see 3 being abused by slimes in a similar fashion to dionas kudzuflooding.
5 is fine, this and 2 are really the only things I think slimes need.
I don't think I understand 6
7 seems like it'll be abused to massive extents, maybe make them take like, 4x6 inventory space. Honestly, I think that picking up and fireman carrying people should be it's own PR entirely. shouldn't be exclusive to slimes, they should just be maybe less heavy so you don't slow down as much when carrying them.

Overall, I would think about 2 and 5 but the rest seem like they would make slimes unbalanced. Especially 1 and 7. (also wouldn't this PR require a design document?)

**Comment by LankLTE** [comment]
> Slimes have an accelerated regen (slight)

What a great idea I wonder why nobody ever thought of this :godo:

**Comment by Just-a-Unity-Dev** [comment]
> IMO this PR just seems to powercreep slimes massively. 1 seems like a very OP ability used exclusively by shitters, to bust into a locked room and shoot someone dead instantly 2 seems fine honestly, it's the species ability I would give to slimes. Just make it so that examining them makes it obvious they have something inside. 3 and 4 are fine, although I can see 3 being abused by slimes in a similar fashion to dionas kudzuflooding. 5 is fine, this and 2 are really the only things I think slimes need. I don't think I understand 6 7 seems like it'll be abused to massive extents, maybe make them take like, 4x6 inventory space. Honestly, I think that picking up and fireman carrying people should be it's own PR entirely. shouldn't be exclusive to slimes, they should just be maybe less heavy so you don't slow down as much when carrying them.
> 
> Overall, I would think about 2 and 5 but the rest seem like they would make slimes unbalanced. Especially 1 and 7. (also wouldn't this PR require a design document?)

...obviously it powercreeps slimes - because slimes have nothing _interesting_ to them. i can link you the issue stating how slimes have purely negatives - #23354, but these are all valid things that apply to a slimeperson

**Comment by Just-a-Unity-Dev** [comment]
also i want 7 in specific because i think it would be funny to carry a slimeperson in your bag at the cost of it taking up everything

**Comment by LankLTE** [comment]
I feel like the morphing thing might be a bit much tbh, but also given you drop all your shit it's kind of completely irrelevant most of the time so I'm down to at least try it. It feels similar yet different to nymphs so it could maybe be interesting, but I agree that being picked up is more a downside than anything and the "morphed" slimes should probably be very similar to normal slimes.

**Comment by Kadeo64** [comment]
> 
> ...obviously it powercreeps slimes - because slimes have nothing _interesting_ to them. i can link you the issue stating how slimes have purely negatives - #23354, but these are all valid things that apply to a slimeperson

I think it's a little bit too much though. Especially 1. That seems like an antagonist ability, not something that, let's be real, 1/8 of the station is gonna be able to do. Just because you drop everything doesn't make giving an entire species All Access a good mechanic (especially because you can just put your PDA in your internal storage anyway). I'm fine with 2 and 5, and 7 seems funny but should probably be part of a full fireman carry mechanic PR

**Comment by Just-a-Unity-Dev** [comment]
i'll drop the breaking and entering but morphs are still gonna be in

**Comment by BreadyToad** [comment]
> > imo, internal inventory and putting into bags is OP
> 
> It's a 2x2 inventory that drops on crit, if necessary I can make it so that people can open the internal inventory - not sure how bags are OP, but I guess I can make it so that only the morph can fit inside of a bag

there is a reason why internal storage is a syndie implant. Giving slimes this would just make all powergaming antags playing slimes. As I already stated, it's better to simply give slimes the mechanics they had in SS13, such as being able to change your hairstyle and beard at wish and drinking dye to change colour (I played only Paradise, so my examples are only based on their mechanics). Additionally, when surgery gets added slimes will be able to regen lost limbs and will be immune to bone fractures, meaning they almost won't slow down at low health compared to other races, which, combined with this PR, will make them completely OP

**Comment by lzk228** [comment]
> have nothing _interesting_ or GOOD to them

they will have when wounds rework is done
like ubreakable bones (they dont have it), regenerate limbs

also would good to be able to change your hairstyle anywhere anytime, for food (hungry = no haircut :trollface:)

**Comment by CrigCrag** [comment]
wounds rework coming soon (2032)

**Comment by Just-a-Unity-Dev** [comment]
> they will have when wounds rework is done
> like ubreakable bones (they dont have it), regenerate limbs

And until wounds is merged they will stay uninteresting - I am not going to rely on "a PR that someone will finish _eventually_", but rather I would like to be the person who finishes the PR.

**Comment by Kadeo64** [comment]
> I feel like the morphing thing might be a bit much tbh, but also given you drop all your shit it's kind of completely irrelevant most of the time so I'm down to at least try it. It feels similar yet different to nymphs so it could maybe be interesting, but I agree that being picked up is more a downside than anything and the "morphed" slimes should probably be very similar to normal slimes.

I can already see shitters breaking into armory and stealing a pistol, then hiding it in their internal storage.

**Comment by Just-a-Unity-Dev** [comment]
> i'll drop the breaking and entering but morphs are still gonna be in

bump

**Comment by Just-a-Unity-Dev** [comment]
updated todo list

**Comment by UbaserB** [comment]
> also i want 7 in specific because i think it would be funny to carry a slimeperson in your bag at the cost of it taking up everything

literally felinids

**Comment by mirrorcult** [comment]
morphing is good and i want it in any form here it'll be really fun

**Comment by Froozigiusz** [comment]
> Where is the tg self-revival mechanic 😢

Once we get the nymph system to dionæ #19450, I mean let us have something with those bipedal shrubs.
Or maybe the #17279 perhaps, they both are quite exciting to see

**Comment by strO0pwafel** [comment]
> > I feel like the morphing thing might be a bit much tbh, but also given you drop all your shit it's kind of completely irrelevant most of the time so I'm down to at least try it. It feels similar yet different to nymphs so it could maybe be interesting, but I agree that being picked up is more a downside than anything and the "morphed" slimes should probably be very similar to normal slimes.
> 
> I can already see shitters breaking into armory and stealing a pistol, then hiding it in their internal storage.

morphing would drop the pistol though

**Comment by Just-a-Unity-Dev** [comment]
> > > I feel like the morphing thing might be a bit much tbh, but also given you drop all your shit it's kind of completely irrelevant most of the time so I'm down to at least try it. It feels similar yet different to nymphs so it could maybe be interesting, but I agree that being picked up is more a downside than anything and the "morphed" slimes should probably be very similar to normal slimes.
> > 
> > 
> > I can already see shitters breaking into armory and stealing a pistol, then hiding it in their internal storage.
> 
> morphing would drop the pistol though

Not if it's in the internal inventory.

**Comment by asperger-sind** [comment]
If this PR is to be finished (rain pr lol lmao) then this should come off as a full-on slime rework ditching or incorporating and reimagining the old bonuses and maluses, introducing new ones in the process, not doing such would put slimes into a position where the features of the species are stitched together from PRs made by different contributors over different times, which is obviously not good because of how slimes were out of sync for a while.

The overall strength of the planned features is greater than of any other species both by quantity and individual feature strength, and as such it should be counteracted by equally strong maluses, I'll go over the planned features and then propose some maluses to counteract them.

Slime morph. Relatively weak for the debuffs it grants, losing all of your equipment in a largely equipment-focused game in exchange for a little speed and the ability to traverse the few grilles in-station while also being liable to getting picked up massively downplays the potentially strong and defining feature, if anything this should allow slimes to slowly crawl under doors that are not "secure", so no sec doors or windoors, no command doors or windoors, no vault, etc.
This should also leave mild amounts of goo on the floor and deal damage to the slime, perhaps make a loud noise as well.

Slime inventory. Effectively a weaker form of what arachnids have with web pockets but overall, a good side feature that doesn't lean heavily into being too strong or weak.

Blood slow immunity. Decent side feature, assuming this applies to blood as well as slime it's not particularly useful except for situations like med, nukies or chef.

Damage resistances. Slime DR should be overhauled to match with the current combat balance, having it as-is won't exactly do much good.

Faster regen. Alright QoL feature assuming it really is only slightly faster.

As such, there are 5 differentiating buffs coming with slimes, I will also propose another feature which is leaning both into buff and debuff now.

Slime goo should be prone to easier detective investigation, DNA being linked directly to slime goo and shown in the crew records console. If a morphed slime goes to atmos, fucks with the settings and obviously doesn't clean up for the lack of mop a detective can show up and go "GOOD GOLLY A SLIME HAS MORPHED INTO HERE FUCKING KILL THIS USER", but this also allows for easier murder identification if someone has been killed.

Slimes should get hit HARD by poison and radiation, if a slime gets poisoned or overdosed their silly slime cells eviscerate and die in agony, same with radiation, both because Slime Logic :tm:

Slimes should be limited to 1 chemical being metabolized at a time, something something slimemen are complex organisms and have a hard time incorporating meds into themselves.

Slime fists should be weaker, like 2 (as opposed to 5) blunt damage with 2 attack speed (gloves override this), attack speed because funny jellyman rapid slapping.

Slimes should have slower melee speed in general, nothing too much, assuming all other species have a modifier of 1.0 slimes should be like 0.85 or 0.9 to avoid overnerfing silly slimemen.

Slime DR should probably be more specialized towards one specific damage type like it was for blunt before but if slimes are despite the above maluses deemed too strong the DR could be adjusted to be weaker than most other species DRs with some careful number maneuvers.

text wall goopman blast, graaaaarrgh..,.,. glug lug glug glug,.,.,.,... (slime noises)

**Comment by K-Dynamic** [comment]
Yeah idk about the 1 chem and weakness to poison/rads thing, that'll make them difficult to treat if poison is still in their system

Also slimes by nature are complex but also walking osmosis bubbles, so they should be able to metabolise many chems at once imo

For flavour, consider adding chems they can reproduce like reagent slimes

**Comment by Just-a-Unity-Dev** [comment]
Added to OP

**Comment by IamVelcroboy** [comment]
Morphing and squeezing through grilles is such a good feature. Would be extra funny if they could fall into or hide in drains.

**Comment by K-Dynamic** [comment]
> Morphing and squeezing through grilles is such a good feature. Would be extra funny if they could fall into or hide in drains.

inb4 squeezing through electrified grille and becoming cooked inside out

**Comment by Just-a-Unity-Dev** [comment]
we're pulling a rainpr - i'll finish this by saturday im really bummed out by life

**Comment by Kadeo64** [comment]
> Morphing and squeezing through grilles is such a good feature. Would be extra funny if they could fall into or hide in drains.

floor drains should act like chasms to slimes

**Comment by Varen** [comment]
What if while morphed, you replaced the unpowered airlock open do after with squeezing through the airlock to the other side instead? Still a do after, still only unpowered, only difference is you're leaving the door closed. But it feels kinda flavourful. Maybe make it just a bit faster but still slower than crowbar opening.

**Comment by Just-a-Unity-Dev** [comment]
I don't want to do anything SS14-related anymore

**Comment by Just-a-Unity-Dev** [comment]
actually it's a dick move to all slime players for me to just close this pr

**Comment by K-Dynamic** [comment]
> actually it's a dick move to all slime players for me to just close this pr

Let it be someone else's problem, don't burn yourself out over this

**Comment by Just-a-Unity-Dev** [comment]
image

**Comment by Just-a-Unity-Dev** [comment]
lol

**Comment by Just-a-Unity-Dev** [comment]
fuck you too tests

**Comment by Krunklehorn** [comment]
I don't play slimes but I skimmed through this PR and would love to see it get merged.

1) The passive health regen is a bit much. I think slimes should be encouraged to make use of their advanced metabolic system rather than receive a recovery boost just for existing.

2) You mentioned at one point they might be able to squeeze under doors? This is a huge security concern, no matter how you balance it. 👎 

Some questions...
3) When morphing, slimes drop their internal inventory, yes?
4) Other players can view and interact with their internal inventory too, yes?

Everything else seems fine. Great work! ❤️

**Comment by Just-a-Unity-Dev** [comment]
1. ok
2. already revised, they don't go under doors
3. ill have to check
4. yea

**Comment by TrapFANchik** [comment]
I will say one thing. You are a saint

**Comment by K-Dynamic** [comment]
Make slimes transform into puddles (i.e. change slime mob sprite to slime puddle sprite) :trollface:

**Comment by Lomovar** [comment]
merge when?

**Comment by notquitehadouken** [comment]
storage implant no virus 2025 working

**Comment by lzk228** [comment]
why is slimeperson grey but slime is blue? its confusing
also, imo all that seems too OP

**Comment by Just-a-Unity-Dev** [comment]
> why is slimeperson grey but slime is blue? its confusing
> also, imo all that seems too OP

Perhaps if you read the sister issue you would understand all of this

**Comment by metalgearsloth** [comment]
> > why is slimeperson grey but slime is blue? its confusing
> > also, imo all that seems too OP
> 
> Perhaps if you read the sister issue you would understand all of this

Half the OP is ranting and most of the comments are bikeshedding so please clarify where it is rather than brushing it off.

**Comment by Just-a-Unity-Dev** [comment]
> Half the OP is ranting and most of the comments are bikeshedding so please clarify where it is rather than brushing it off.

Slimes are not fun to play as and they also lack any neat features, they are more or less just translucent colored humans with worse damage mods and no interesting gameplay features. This is why I think they're the worst race in the game.

"but dionae were designed to be the worst!"
Okay, but at least dionae have interesting enough features like having Robust Harvest act as omnizine, and even have a *design*. Slimes don't have any of those, instead just being nitrogen breathers and being able to metabolize 6 chems at once.

File: Resources/Prototypes/Entities/Mobs/NPCs/slimes.yml
Comment (metalgearsloth) [inline Resources/Prototypes/Entities/Mobs/NPCs/slimes.yml:153]: Slimes shouldn't be able to break out of perma as you'd have to redesign every single map.

File: Resources/Prototypes/Entities/Mobs/Species/slime.yml
Comment (metalgearsloth) [inline Resources/Prototypes/Entities/Mobs/Species/slime.yml:56]: Healing the 2 most common damage types in the game are kinda sus.

File: Content.Server/Geras/GerasComponent.cs
Comment (metalgearsloth) [inline Content.Server/Geras/GerasComponent.cs:10]: Unfinished.

File: Content.Server/Geras/GerasComponent.cs
Comment (metalgearsloth) [inline Content.Server/Geras/GerasComponent.cs:18]: Needs to be a datafield or it breaks persistence.

File: Resources/Prototypes/Entities/Mobs/NPCs/slimes.yml
Comment (metalgearsloth) [inline Resources/Prototypes/Entities/Mobs/NPCs/slimes.yml:136]: Remove the comment

**Review by metalgearsloth — CHANGES_REQUESTED** [review]
Needs predicting.

The ability to break out of perma easily seems extremely overpowered and will just lead to slimes getting executed.

Also not sure how this isn't just better human now.

**Comment by Just-a-Unity-Dev** [comment]
Mini slimes don't go under doors. They aren't better human because they still have their drawbacks of being weak to piercing, slash, alongside their reduced walkspeed and attack speed (I'm fairly sure I coded that)

**Comment by Just-a-Unity-Dev** [comment]
Also I'm not sure how to make polymorphing predicted

**Comment by wafflewarior** [comment]
I really hope to see this merged. as an avid slime player I really like the idea of being able to revert to slime and scurry around and being able to have a sort of bonus pocket will be very nice. I hope to be stuffed in a bag someday

File: Resources/Prototypes/Entities/Mobs/NPCs/slimes.yml
Comment (Just-a-Unity-Dev) [reply Resources/Prototypes/Entities/Mobs/NPCs/slimes.yml:136]: oops lmao

**Comment by Just-a-Unity-Dev** [comment]
Still working on this.

**Comment by XxSWAG-MASTER-OVERLORDxX** [comment]
I do hope this comes soon, considering the recent slime "nerf" if you can call it that.

File: Resources/Prototypes/Entities/Mobs/Species/slime.yml
Comment (Just-a-Unity-Dev) [reply Resources/Prototypes/Entities/Mobs/Species/slime.yml:56]: To be fair, they are also weak to all of afformentioned damage types - but sure, I'll raise the damage cap

**Comment by Just-a-Unity-Dev** [comment]
Reviews done

File: Resources/Prototypes/Entities/Mobs/Species/slime.yml
Comment (Just-a-Unity-Dev) [reply Resources/Prototypes/Entities/Mobs/Species/slime.yml:27]: i tested it with stripping (right click->stripping) and it seemed to be fine??? citation needed, but im sure its fine

**Review by mirrorcult — APPROVED** [review]

**Comment by FairlySadPanda** [comment]
SLIMEPEOPLE
YOUR HOUR COMETH

**Comment by Clonko** [comment]
On one hand we ball. On the other we cant be revived, nor healed by traditional means.

**Comment by Just-a-Unity-Dev** [comment]
known issue, it's due to internal inventory - its late at my place unfortunately so if someone could disable internal inventory temporarily until i find out a solution it would be much appreciated 😁😁

**Comment by LordEclipse** [comment]
> known issue, it's due to internal inventory - its late at my place unfortunately so if someone could disable internal inventory temporarily until i find out a solution it would be much appreciated 😁😁

Just have interactions take priority over opening inventory. Or just leave internal inventory opening to the context menu.

**Comment by Sworple** [comment]
clicking yourself to inject with, for example, an uplink implant will simply try to put it into your internal inventory, with no override to simple interactions meaning slimepeople are locked out of using implants without someone helping them, most likely.

**Pull request merged**
